systemjs/webpack-externals/webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+
+// See https://github.com/joeldenning/systemjs-webpack-interop
+const systemjsInterop = require('systemjs-webpack-interop')
+
+module.exports = systemjsInterop.modifyWebpackConfig({
```

Comment:
I'm open to removing the systemjs-webpack-interop lib and replacing with just a raw webpack config